config/application.rb

```diff
@@ -154,6 +154,10 @@ class Application < Rails::Application
       }
     }
 
+    config.mediator_contact_info = {
+
```

Comment:
@cbeer, can you clarify what my to-do is here? Provide the mediator email addresses?